packages/publisher/github/src/PublisherGithub.ts

```diff
@@ -1,7 +1,12 @@
+import path from 'path';
```

Comment:
We started doing this a while back right? We should add a lint rule for it.

```suggestion
import path from 'node:path';
```